webrtc.html

```diff
@@ -4685,7 +4685,7 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
         <li>
           <p>If the <a href="#dfn-underlying-data-transport">transport</a> was
           closed <dfn id="data-transport-closed-error">with an error</dfn>,
-          fire an NetworkError event at <var>channel</var>.</p>
+          queue a task to fire an NetworkError event at <var>channel</var>.</p>
```

Comment:
Based on comment from @Jan-Ivar and feedback from the Mozilla team, I will remove the specific change from the proposal.  @adam-be, it might be better to open a new issue to address the question on the NetworkError event.